CHANGELOG.md

```diff
@@ -10,7 +10,11 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ### Oneview Features supported
 - Enclosure
 - Enclosure Group
+- Logical Enclosure
+- Server Hardware
+- Server Hardware Template
```

Comment:
I can see files changed for Server Profile and Server Profile Template. Please change resource names here.